OWNERS

```diff
@@ -0,0 +1,6 @@
+assignees:
```

Comment:
This seems to be inconsistent everywhere (core uses approvers and reviewers)... I'm following the example in the template linked from the incubation doc: https://github.com/kubernetes/kubernetes-template-project/blob/master/OWNERS